content/md/products-super-power/understand-data-quality.md

```diff
@@ -0,0 +1,88 @@
+---
+id: understand-data-quality
+themes: products-super-power, first-steps
+title: Understand **data quality**
+popular: false
+ee-only: true
+related:
+---
+
+# About data quality
+What is this **red, orange or green A, B, C, D, E displayed on my product edit form**? What are the **enrichment** and **consistency** grades?
+
+**This is data quality insights!**
+
+Because rich, coherent and consistent product data is important to you, we introduced the concept of "data quality" in our PIM.
+Akeneo's data quality is an information about your product data quality: it helps you measure your data quality on two axis: **enrichment** and **consistency**.
+Each of these two axis has a grade ranging from A to E. The grades can be seen on several screens of the PIM.
+
+![Image data quality badges](../img/data-quality-grades.png)
+
+# How is enrichment calculated?
+The **enrichment grade** is calculated on the percentage of attributes filled in your product edit form (whether the attributes are required for completeness or not).
+
+Checking the `data quality insights` panel in the product edit form allows you to quickly see which attributes need to be filled to improve your data quality grade for enrichment.
+
+::: tips
+From the `data quality insights` panel, by clicking on the "Display all missing attributes", you can easily access a product edit form filtered on all the attributes that need to be filled.
+:::
+
+# How is consistency calculated?
+
+The **consistency grade** is calculated on several criteria. The mix of the following criteria help the PIM determine the grade for consistency:
+- Spell check: text and text areas are checked for spelling mistakes.
+- Title formatting: the attribute as main title is checked for optimised writing.
+- Upper cases formatting: the PIM checks whether text area attributes are correctly formatted for upper cases.
+- Lower cases formatting: the PIM checks whether text area attributes are correctly formatted for lower cases.
+
+::: info
+Spell check is available for English, French, German and Spanish locales.
+Spell check is usable with text and simple text area attributes.
```

Comment:
"simple text are" means without the WYSIWYG enabled right?

Reply:
yes

Reply:
on lines 35 and 36, normally one uses "uppercase" and "lowercase" as one word when it is an adjective or a noun -- but two words isn't wrong. And, they are not normally plural, so it should be "uppercase formatting" or "upper case formatting."

Reply:
Thnks @jmevans00 I asked Camille about this and she said both could be used as well. Which one do you suggest we use to be most universal? I'll go with your pick ;-)

Reply:
my suggestion is "uppercase formatting"